enterprise-suite/install-es.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
+
+set -eu
+
+function docvar() {
+    envvar=$1
+    local detail=$2
+    eval echo "$envvar=\$${envvar}"
+    if [ -n "$detail" ]; then
+        echo -e "\t $detail"
+    fi
+    echo
+}
+
+function usage() {
+    echo "$0 [-h]"
+    echo
+    echo "Environment variables that can be set:"
+    echo
+    docvar LIGHTBEND_COMMERCIAL_CREDENTIALS "Credentials file in property format with username/password"
+    docvar ES_REPO "Helm chart repository"
+    docvar ES_CHART "Chart name to install from the repository"
+    docvar ES_NAMESPACE "Namespace to install ES-Console into"
+    docvar ES_MINIKUBE "Set to true to enable minikube specific options"
+    docvar ES_LOCAL_CHART "Set to location of local chart tarball"
+    docvar ES_UPGRADE "Set to true to perform a helm upgrade instead of an install"
+    docvar ES_VERSION "Set to non-empty to install a specific version"
+    exit 1
+}
+
+function export_credentials() {
+    # Exported variables.
+    repo_username=
+    repo_password=
+
+    # Determine credentials
+    if [[ -n "$LIGHTBEND_COMMERCIAL_USERNAME" && -n "$LIGHTBEND_COMMERCIAL_PASSWORD" ]]; then
+        repo_username=$LIGHTBEND_COMMERCIAL_USERNAME
+        repo_password=$LIGHTBEND_COMMERCIAL_PASSWORD
+    elif [ -e $LIGHTBEND_COMMERCIAL_CREDENTIALS ]; then
+        while IFS='=' read -r key value; do
+            local trimmed_key=$(echo -e "$key" | tr -d '[:space:]')
+            local trimmed_value=$(echo -e "$value" | tr -d '[:space:]')
+            if [ "$trimmed_key" == "user" ]; then
+                repo_username=$trimmed_value
+            fi
+            if [ "$trimmed_key" == "password" ]; then
+                repo_password=$trimmed_value
+            fi
+        done < $LIGHTBEND_COMMERCIAL_CREDENTIALS
+    fi
+
+    if [[ -z "$repo_username" || -z "$repo_password" ]]; then
+        echo "Credentials missing, please check your credentials file"
+        echo "LIGHTBEND_COMMERCIAL_CREDENTIALS=${LIGHTBEND_COMMERCIAL_CREDENTIALS}"
+        echo
+        usage
+    fi
+}
+
+# User overridable variables.
+LIGHTBEND_COMMERCIAL_USERNAME=${LIGHTBEND_COMMERCIAL_USERNAME:-}
+LIGHTBEND_COMMERCIAL_PASSWORD=${LIGHTBEND_COMMERCIAL_PASSWORD:-}
+LIGHTBEND_COMMERCIAL_CREDENTIALS=${LIGHTBEND_COMMERCIAL_CREDENTIALS:-$HOME/.lightbend/commercial_credentials}
+ES_REPO=${ES_REPO:-https://lightbend.github.io/helm-charts}
+ES_CHART=${ES_CHART:-enterprise-suite}
+ES_NAMESPACE=${ES_NAMESPACE:-lightbend}
+ES_MINIKUBE=${ES_MINIKUBE:-false}
```

Comment:
I should change enterprise-suite/Makefile:75 to use this style...

Reply:
I don't think this works in `make`, it's bash specific.